src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -152,17 +152,14 @@ private Entity getLectureEntityFromRequest(HttpServletRequest request) {
   }
 
   /**
-   * Returns URL redirecting to lecture view page with parameters {@code lectureId}
-   * and {@code videoId} found in {@code lectureEntity}.
+   * Returns URL redirecting to lecture view page with parameter {@code lectureId}
```

Comment:
Please explain `lectureId` in the comment, or remove.

I think: `Returns URL for the lecture view page for {@code lectureEntity}` might be good enough

Reply:
Fixed